test_conformance/math_brute_force/unary_float.cpp

```diff
@@ -435,7 +435,7 @@ cl_int Test(cl_uint job_id, cl_uint thread_id, void *data)
 
                 if (fail)
                 {
-                    if (ftz)
+                    if (ftz || relaxedMode)
```

Comment:
There are a bunch of other places with `if (ftz)` checks.  Should they be updated too?

Reply:
Yes. Just wanted to get some eyes before I can proceed further. 
If the proposed fix looks good, I can update the other files too. 
This is likely to increase runtime in some cases as this possibly is going to require checking against two reference computations (Check against the default first and then against the relaxed reference computation.

Reply:
Have updated at other places too. Let me know if you have any further suggestions.